docs/content/formatting.stories.mdx

```diff
@@ -149,3 +149,15 @@ When a date range overlaps a year, make sure the year is indicated.
 | ✅ **Do**                                        | ❌ **Don't**                           |
 | ------------------------------------------------ | -------------------------------------- |
 | Dec 31 2022, 2:00 PM &#8211; Jan 1 2023, 3:00 PM | Dec 31, 2:00 PM &#8211; Jan 1, 3:00 PM |
+
+<br />
+
+### Numerical abbreviation
+
+Minimize the use of decimal places in numbers unless essential. Begin abbreviating numbers at ten thousand when precision is not crucial.
```

Comment:
Would it be helpful to provide an example of when decimals could be essential? (tabular data maybe?) Let me know what you think.